File: src/main/java/com/ngolajr/ispmedia/entities/Video.java
package com.ngolajr.ispmedia.entities;

import jakarta.persistence.Entity;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Entity
@ToString
@Getter
@Setter
public class Video extends Conteudo{
    private String mimeType;
    private long duration;
    private String path;
    private int streams;
    private String formato;
    private String tamanho;
    private String type;
    public Video(){
        super();
        this.streams = 0;
    }
}

File: ISPMEDIA-ANGULAR/src/app/entities/Video.ts
import {Conteudo} from "./Conteudo";
import {Genero} from "./Genero";
import {User} from "./User";

export class Video extends Conteudo{
  duration:number = 0;
  path:string = '';
  streams:number = 0;

  constructor(id: string, titulo: string, thumbNailUri: string,
              descricao: string, genero:Genero, editora:string, user:User) {
    super(id, titulo, thumbNailUri, descricao, genero, editora, user);
  }
}
